modules/ldap/main.go

```diff
@@ -384,6 +407,18 @@ func createUser(req nano.Request) (*nano.Response, error) {
 		}), err
 	}
 
+	exists, err := CheckUserExists(params.UserEmail, ldapConnection)
+	if err != nil {
+		return nano.JSONResponse(500, hash{
+			"error": err.Error(),
+		}), nil
+
+	}
+	if exists {
+		return nano.JSONResponse(403, hash{
```

Comment:
Hi,

403 is not the correct code to return here.
I think about 409 (conflict).